### ISSUE TITLE
Change newznab to use tvdb instead of tvrage

### ISSUE THREAD

**Issue by fernandog** [SiCKRAGETV/sickrage-issues#2748]
@miigotu I send you an email from nzbgeek admin.

We need to think how to solve this as not all providers have fully tvdb support

https://github.com/SiCKRAGETV/SickRage/blob/develop/sickbeard/providers/newznab.py#L407

**Comment by miigotu** [comment]
@fernandog I already removed the rid param. rageid is always 0 there.

**Comment by Taloth** [comment]
In all likeliness nn+ will replace ```rid=``` with ```tvid=``` or ```tvdbid=``` (undecided).
In addition there's a chance that the caps endpoint will also return a ```supportedParams``` attribute on the search elements. Like:
```xml
<searching>
  <search available="yes" supportedParams="q" />
  <tv-search available="yes" supportedParams="q,tvdbid,season,ep" />
  <movie-search available="no" supportedParams="q" />
</searching>
```
That ```supportedParams``` attrib is merely a proposal to the nn+ devs, I'll let you know if they decide something definitive. But if they approve then you'll be able to determine whether an indexer supports tvdbid searching by querying (and caching) t=caps.

**Comment by miigotu** [comment]
@Taloth This is already in the works. I have been speaking to dognzb admin who has been relaying information to me and to other nn+ indexers. AFAIK tvdbid was decided as the new param because tvid is too generic.

Supported params would be helpful (although it makes me have to rewrite more code xD)

**Comment by Taloth** [comment]
Awesome, wasn't sure if you guys were kept in the loop as well.

We devs love writing code :) fortunately I did most of the work already for torznab, just had to add tvdbid.

**Comment by miigotu** [comment]
Preliminary changes merged into develop in SiCKRAGETV/SickRage#2600
